src/Uno.UI/UI/Xaml/Media/Animation/Animators/GPUFloatValueAnimator.iOSmacOS.cs

```diff
@@ -421,12 +441,23 @@ private UnoCoreAnimation InitializeCompositeCoreAnimation(CompositeTransform tra
 			=> CreateCoreAnimation(transform.View, property, nsValueConversion, transform.StartAnimation, transform.EndAnimation);
 
 		private UnoCoreAnimation CreateCoreAnimation(
-			_View view,
+			_View? view,
 			string property,
 			Func<float, NSValue> nsValueConversion,
-			Action prepareAnimation = null,
-			Action endAnimation = null)
+			Action? prepareAnimation = null,
+			Action? endAnimation = null)
 		{
+
+			if (view is null)
+			{
+				if (this.Log().IsEnabled(Uno.Foundation.Logging.LogLevel.Warning))
+				{
+					this.Log().Warn("The View property of the Transform is null.");
+				}
+
+				throw new InvalidOperationException("The View property of the Transform is null.");
+			}
```

Comment:
If we are disallowing null here, I think the parameter shouldn't be annotated as nullable, and callers should make sure to not pass null

Reply:
Updated